src/pages/images/actions/forms/ImportImageFromTarballForm.tsx

```diff
@@ -0,0 +1,146 @@
+import { ChangeEvent, FC, useState } from "react";
+import { useEventQueue } from "context/eventQueue";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import {
+  changeImagePublicity,
+  createImageAlias,
+  createImageFromTarball,
+} from "api/images";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { Link, useNavigate } from "react-router-dom";
+
+interface Props {
+  close: () => void;
+  project: string;
+}
+
+const ImportImageFromTarballForm: FC<Props> = ({ close, project }) => {
```

Comment:
we should implement progress status for file upload similar to how it's done in `UploadCustomIso`. @edlerd I see we are using `axios` for this, although is it possible to track multiple file uploads at the same time (for the split tarball import)?

Reply:
@edlerd , should we implement an axios approach for this?

Reply:
I think axios is a nice helper to get the upload status. We can have a progress bar per file that is being uploaded, or restart the same progress bar for each new file.